src/addDebouncedHandler.tsx

```diff
@@ -0,0 +1,42 @@
+import {addHandlers, UnchangedProps, flowMax} from 'ad-hok'
+import {debounce} from 'lodash'
+
+import addPropTrackingRef from './addPropTrackingRef'
+import cleanupProps from './cleanupProps'
+
+type AddDebouncedHandlerType = <
+  TPropName extends string,
+  TProps extends {[handlerName in TPropName]: (...args: any[]) => any}
```

Comment:
I "reversed" the typical pattern of eg `<TProps, TPropName extends keyof TProps>` in order to try and further refine it to not just say "the argument is a prop name" but "the argument is a prop name of a handler"

It seems to work and does flag when you pass a prop name of something other than a handler, but the Typescript error is less clear than with the normal pattern (it seems to be flagging the *preceding* step in the chain), so might be better to revert this to the normal pattern